satellite/satellitedb/testdata/postgres.v70.sql

```diff
@@ -358,7 +358,7 @@ CREATE UNIQUE INDEX serial_number ON serial_numbers ( serial_number );
 CREATE INDEX serial_numbers_expires_at_index ON serial_numbers ( expires_at );
 CREATE INDEX storagenode_id_interval_start_interval_seconds ON storagenode_bandwidth_rollups ( storagenode_id, interval_start, interval_seconds );
 
-CREATE UNIQUE INDEX credits_earned_user_id_offer_id ON user_credits (id, offer_id) WHERE credits_earned_in_cents=0;
+CREATE UNIQUE INDEX credits_earned_user_id_offer_id ON user_credits (id, offer_id);
```

Comment:
Yessss the sooner we get this in, the better, so no one else keeps copying the broken version into new files

Reply:
lol! i know, its so annoying!